### PR TITLE
About us page content

### PR DESCRIPTION
**Pull request by triszt4n**
# What's new:

* Content created for about us page.

### CODE REVIEW

File: src/components/about-components/AboutParagraph.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Box, Heading, Link } from '@chakra-ui/react'
+import React from 'react'
+
+interface ParagraphProps {
+  id?: string
+  title: string | undefined
+  titleSize: string | undefined
```
Comment (OmTheTurtle) [inline src/components/about-components/AboutParagraph.tsx:7]: Why not use the `?` here like on `id`?

Comment (triszt4n) [reply src/components/about-components/AboutParagraph.tsx:7]: True, very true. I must've used quick context actions :D

**Comment by OmTheTurtle** [comment]
Can this be deployed to somewhere (Vercel maybe)? It'd be much easier to review the text content than traversing the source code.

**Comment by triszt4n** [comment]
Yep working on it, Netlify is lazy listening to my commits :sleeping:

File: src/pages/about.tsx [hunk 2]
```diff
@@ -1,82 +1,336 @@
...
+                  Fő feladatunk a{' '}
+                  <GatsbyLink to="/projects/pek-next">
+                    <Text display="inline" w="full" bgClip="text" bgGradient="linear(to-r, tomato, orange.300)" fontWeight="extrabold">
+                      Profil és Körök
+                    </Text>
+                  </GatsbyLink>{' '}
+                  – röviden PéK – folyamatos fejlesztése és karbantartása. Jelenleg ezen az alkalmazáson keresztül folyik a kar közösségi
+                  pontozása. Ez a rendszer már több generációt is megélt az aktív körtagoknak köszönhetően: legelőször Kollégiumi
+                  Információs Rendszernek hívtuk (KIR), aztán a kiterjedésével Villanykari Információs Rendszer (VIR) lett, amelynek tech
+                  stackjében részt vett a JavaEE és az OpenAM. A PéK mai nevét ????-ben vette fel, korábbi generációja még JavaEE-s alapokon
```
Comment (triszt4n) [inline src/pages/about.tsx:82]: "A PéK mai nevét ????-ben vette fel," <-- I should've left this bit out 😅

**Comment by triszt4n** [comment]
Will get resolved by #67

**Pull request closed without merging**